app/admin/microcredit.rb

```diff
@@ -242,3 +243,27 @@ def update
   end  
 
 end
+
+ActiveAdmin.register MicrocreditOption do
+  menu false
+  belongs_to :microcredit
+  navigation_menu :default
+
+  permit_params :microcredit_id, :location, :parent
+
+  form partial: "microcredit_option", locals: { spain: Carmen::Country.coded("ES") }
```

Comment:
I think that `locals` is not needed here

Reply:
ok, removing locals part.